packages/discord.js/src/structures/Emoji.js

```diff
@@ -56,6 +56,17 @@ class Emoji extends Base {
     return this.id && this.client.rest.cdn.emoji(this.id, this.animated ? 'gif' : 'png');
   }
 
+  /**
+   * @type {?string}
+   * @param {ImageExtension} ext the extension of emoji
+   * @returns {string}
+   */
+  getUrl(ext) {
+    const extensions = ['webp', 'png', 'jpg', 'jpeg', 'gif'];
+    if (!extensions.includes(ext) || !ext) return undefined;
+    return this.id && this.client.rest.cdn.emoji(this.id, ext);
```

Comment:
I think it's best to throw an error when the user provides an invalid extension.

Reply:
you are right i will do it